multiset.go

```diff
@@ -0,0 +1,97 @@
+package secp256k1
+
+// #include "./depend/secp256k1/include/secp256k1_multiset.h"
+import "C"
+import "github.com/pkg/errors"
+
+// MultiSet is a type used to create an Elliptic Cure Multiset Hash.
+// which is a rolling(homomorphic) hash that you can add and remove elements from
+// and receiving the same resulting hash as-if you never hashed that element.
+// Because of that the order of adding and removing elements doesn't matter.
+// The type should only be created via the supplied methods.
+type MultiSet struct {
+	set C.secp256k1_multiset
+}
+
+// NewMultiset return an empty initialized set.
+// when finalized it should be equal to a finalized set with all elements removed.
+func NewMultiset() MultiSet {
+	multiset := MultiSet{}
+	ret := C.secp256k1_multiset_init(context, &multiset.set)
+	if ret != 1 {
+		panic("failed intializing a multiset. Should never happen")
+	}
+	return multiset
+}
+
+// Reset the multiset. Equivalent to creating a new empty set.
```

Comment:
The correct way to write go comments over exported functions is "[Function Name] does blablabla".
A.k.a. "Reset clears the multiset from all data. Equivalent to creating a new empty set."

Note: I've seen similar stuff in many Functions, commenting only here.

Reply:
Thanks